packages/web/src/app/[lng]/(auth)/memos/_components/ExtensionInstallCheckDialog/index.tsx

```
@@ -0,0 +1,67 @@
+  Button,
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from '@src/components/ui';
+import { LanguageType } from '@src/modules/i18n';
+
+import {
+  checkLocalStorageKey,
```

ESLint

[error] 'checkLocalStorageKey' is defined but never used.

(unused-imports/no-unused-imports)